### PR TITLE
added issue templates

### LINKED ISSUE #1757: More Issue Templates
image

We have the above issue templates as of now. Adding the following issue templates might be helpful:
- Feature Request
- FTO
- A default template instead of a blank one.
Any suggestions? Please :point_down: :upside_down_face: 

References:
- [Github templates docs](https://docs.github.com/en/enterprise/2.16/user/github/building-a-strong-community/about-issue-and-pull-request-templates)
- [Templates in this project](https://github.com/publiclab/image-sequencer/tree/main/.github/ISSUE_TEMPLATE)

**NOTE**: The FTO template is [here](https://github.com/publiclab/image-sequencer/blob/main/.github/fto-template.md), it just needs to be moved. (And any changes if required)

### PR DESCRIPTION
**Pull request by unnati914**
Fixes #1757   (<=== Replace `0000` with the Issue Number)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm run test-all`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] ask `@publiclab/is-reviewers` for help, in a comment below
* [ ] Insert-step functionality is working correct as expected.
> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
Please make sure to get at least two reviews before asking for merging the PR as that would make the PR more reliable on our part
Thanks!

### CODE REVIEW

**Review by jywarren — APPROVED** [review]

**Review by harshkhandeparkar — COMMENTED** [review]
Why are the files added twice?

**Review by jywarren — CHANGES_REQUESTED** [review]
Just saw the extra copies, but should be pretty easy to remove them! If you'd like help we're here, too!

Also, if it's possible to add `_` characters instead of spaces into the file names, that would be awesome. Thank you!!

File: enhancement template.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# New Enhancement Request
```
[the remark this answers is not in the document]
Comment (unnati914) [reply enhancement template.md:1]: i don't know how this happened will surely remove this

[the remark this answers is not in the document]
Comment (unnati914) [reply enhancement template.md]: done

**Review by jywarren — APPROVED** [review]

**Comment by jywarren** [comment]
Yay, thank you! This is great.

**Pull request merged**